rent_golem/backend/rent_golem/accounts/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+from .models import User
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['email', 'service_password', 'is_staff', 'is_active', 'date_joined']
+
```

Comment:
I think we don't need such a serializer. This file should be probably removed.

Reply:
Deleted